src/Matrix.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+#include <vector>
+#include "ImageWrapper.h"
+
+class matrix
+{
+
+public:
+
+	//constructors
+	matrix();
+
```

Comment:
Maybe add a destructor where you deallocate the array stored as m_doubleptr I believe?

delete[] m_doubleptr;